server/controllers/activitypub/inbox.js

```diff
@@ -20,8 +26,14 @@ const sanitization = {
   }
 }
 
-const controller = async params => {
-  return inboxActivityTypes[params.type](params)
+const controller = async (params, req) => {
+  const { type } = params
+  if (ignoredTypes.includes(type)) {
```

Comment:
```suggestion
  if (!Object.keys(inboxActivityTypes).includes(type)) {
```

Reply:
I prefer having an explicit list of ignored type, it means "we know that type but explicitly decide to not act on it", which is different from how an unknown type, already standardized or in a future extension, should be treated and reported

Reply:
Maybe that distinction isn't that important as both aren't supported anyway. Then we could just reject unknown/unsupported activity types at sanitization (before verifying the signature) with a 400: implementation proposed in 27d1bb8